packages/splitter/package.json

```diff
@@ -21,7 +21,8 @@
   "types": "dist/typings/index.d.ts",
   "dependencies": {
     "@babel/runtime": "^7.8.4",
-    "@zendeskgarden/container-utilities": "^0.7.0"
+    "@zendeskgarden/container-utilities": "^0.7.0",
+    "react-uid": "^2.3.1"
```

Comment:
In order to prevent multi-version dependency bloat, it would be good if this matched `^2.2.0` used by the other container packages.

Reply:
Fixed in https://github.com/zendeskgarden/react-containers/pull/405/commits/04b21508933cf7ae55b3b61e795cb2c365789196